### PR TITLE
[MRG+1] FIX: glass brain plotting to show colorbar when having NaN

### LINKED ISSUE #1922: plot_glass_brain fails to show the colorbar when images have np.inf
Code to reproduce:

```python
import nibabel
from nilearn import plotting
from nilearn.plotting.tests.test_img_plotting import _generate_img

img = _generate_img()
data = img.get_data()

data[6, 5, 2] = np.inf
img = nibabel.Nifti1Image(data, np.eye(4))
plotting.plot_glass_brain(img, colorbar=True)
```
Whereas it is not the case:

- when ```colorbar=False``` with plot_glass_brain
- with plot_stat_map with both colorbar= True or False

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Fixes #1922

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
Good fix! Thanks @KamalakerDadi 

+1 for merge.

**Comment by kchawla-pi** [comment]
Cool. Should we wait until we have results from your PR to reduce
iterations ?  @GaelVaroquaux

**Comment by GaelVaroquaux** [comment]
> Cool. Should we wait until we have results from your PR to reduce
> iterations ? @GaelVaroquaux

How about we let the CI run on this PR to see if it finishes. I'm still
trying to figure out for sure what is going on.

**Comment by KamalakerDadi** [comment]
>How about we let the CI run on this PR to see if it finishes. I'm still
trying to figure out for sure what is going on.

+1

**Comment by kchawla-pi** [comment]
Wow, your build passed in < 2hrs! You have magic Kamalaker!
Add an entry in whats_new when you get the chance.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -1,3 +1,12 @@
+0.5.2
+=====
+
+Fixes
+-----
+
+- `nilearn.plotting.plot_glass_brain` with colorbar=True does not crash when
```
Comment (GaelVaroquaux) [inline doc/whats_new.rst]: Can you add :func: here.

Comment (KamalakerDadi) [reply doc/whats_new.rst]: oops done.

**Comment by GaelVaroquaux** [comment]
Thx! Merging!

**Pull request merged**